torch/autograd/__init__.py

```diff
@@ -50,7 +51,15 @@ def _make_grads(outputs, grads):
     return tuple(new_grads)
 
 
-def backward(tensors, grad_tensors=None, retain_graph=None, create_graph=False, grad_variables=None):
+_TensorOrTensors = Union[torch.Tensor, Sequence[torch.Tensor]]
```

Comment:
This one may be worth moving to `torch.types` when we add that (soon).

Reply:
I'll go ahead and add this right now